lasertagger/bert/tokenization.py

```diff
@@ -161,11 +161,15 @@ def whitespace_tokenize(text):
 class FullTokenizer(object):
   """Runs end-to-end tokenziation."""
 
-  def __init__(self, vocab_file, do_lower_case=True):
+  SPECIAL_SYMBOL_LIST = [".", ",", ";", "!", "?", ":", 
+                               "##.", "##,", "##;", "##!", "##?", "##:"]
+  
+  def __init__(self, vocab_file, do_lower_case=True, do_masking=False):
```

Comment:
s/do_lower_case/lower_case
s/do_masking/enable_mask

avoid "do*" in general which is not common in google codebase.

Reply:
Done.